18F/identity-idp
# Update devDependencies to resolve security advisories

## 🛠 Summary of changes

Updates `msw` and `chai` to resolve security advisories in subdependencies:

- https://github.com/18F/identity-idp/security/dependabot/61
- https://github.com/18F/identity-idp/security/dependabot/62

## 📜 Testing Plan

1. Check that tests which rely on updated dependencies are unaffected
   - `yarn test`
2. Check that security advisories are resolved
   - `yarn audit` does not contain results except as being resolved separately in #9341
